controllers/provider-azure/charts/internal/azure-infra/values.yaml

```diff
@@ -26,6 +26,7 @@ networks:
 outputKeys:
   resourceGroupName: resourceGroupName
   vnetName: vnetName
+  vnetResourceGroup: vnetResourceGroupName
```

Comment:
```suggestion
# vnetResourceGroup: vnetResourceGroupName
```

From your code in the infrastructure actuator it looks as if this output key is optional and not always needed, especially not if a new VNet is created. So, make it optional here as well.

Reply:
done.